tuned/plugins/plugin_net.py

```diff
@@ -12,7 +12,142 @@
+	====
+	+
+	The [option]`features` option allows changing 
+	the offload parameters and other features for the specified
+	network devices. To query the features of your network device,
+	use `ethtool -k device`. The syntax of the option is the same as
+	the [option]`coalesce` option.
+	+
+	.Turn off TX checksumming, generic segmentation and receive offload 
+	====
+	----
+	[net]
+	features=tx off gso off gro off
+	----
+	====
+	The [option]`pause` option allows changing the pause parameters for
+	the specified network devices. To query the pause parameters of your
+	network device, use `ethtool -a device`. The syntax of the option
+	is the same as the [option]`coalesce` option.
+	+
+	.Disable autonegotiation
+	====
+	----
+	[net]
+	pause=autoneg off
+	----
+	====
+	+
+	The [option]`ring` option allows changing the rx/tx ring parameters
+	for the specified network devices. To query the ring parameters of your
+	network device, use `ethtool -g device`. The syntax of the option
+	is the same as the [option]`coalesce` option.
+	+	
+	.Change the number of ring entries for the Rx/Tx rings to 1024/512 respectively
+	=====
+	-----
+	[net]
+	ring=rx 1024 tx 512
+	-----
+	=====
+	+
+	The [option]`channels` option allows changing the numbers of channels
+	for the specified network device. A channel is an IRQ and the set
+	of queues that can trigger that IRQ. To query the channels parameters of your
+	network device, use `ethtool -l device`. The syntax of the option
+	is the same as the [option]`coalesce` option.
+	+
+	.Set the number of multi-purpose channels to 16
+	=====
+	-----
+	[net]
+	channels=combined 16
+	-----
+	=====
+	+   
+	A network device either supports rx/tx or combined queue
+	mode. The [option]`channels` option automatically adjusts the
+	parameters based on the mode supported by the device as long as a
+	valid configuration is requested.
+	+
+	The [option]`nf_conntrack_hashsize` option sets the size of the hash
+	table which stores lists of conntrack entries by writing to
+	`/sys/module/nf_conntrack/parameters/hashsize`.
+	+
+	.Adjust the size of the conntrack hash table
+	====
+	----
+	[net]
+	nf_conntrack_hashsize=131072
+	----
+	====
+	+
+	The [option]`txqueuelen` option allows changing txqueuelen (the length
+	of the transmit queue). It uses `ip` utility that is in package	iproute
+	recommended for TuneD, so the package needs to be installed for its correct
+	functionality. To query the txqueuelen parameters of your network device,
+	use `ip link show` and current value is shown after qlen column.
+	+
+	.Adjust the length of the transmit queue
+	====
+	----
+	[net]
+	txqueuelen=5000
+	----
+	====
+	+
+	The [option]`mtu` option allows changing mtu (Maximum Transmission Unit).
+	It uses `ip` utility that is in package	iproute recommended for TuneD, so
+	the package needs to be installed for its correct functionality. To query
+	the mtu parameters of your network device, use `ip link show` and current
+	value is shown after mtu column.
```

Comment:
`s/after mtu column/after the MTU column/` ? (technical writer will know better)